example/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const createExpoWebpackConfigAsync = require('@expo/webpack-config');
+const path = require('path');
+// Expo CLI will await this method so you can optionally return a promise.
+module.exports = async function (env, argv) {
+  const config = await createExpoWebpackConfigAsync(env, argv);
+  // If you want to add a new alias to the config.
+  config.resolve.alias['react'] = path.join(__dirname, 'node_modules', 'react');
```

Comment:
I think we can add `react-native` and `react-native-dom` aliases here as well like in [Zeego](https://github.com/nandorojo/zeego/blob/master/examples/expo/webpack.config.js#L21):

```suggestion
  Object.assign(config.resolve.alias, {
    react: path.join(__dirname, 'node_modules', 'react'),
    'react-native': path.join(__dirname, 'node_modules', 'react-native-web'),
    'react-native-web': path.join(__dirname, 'node_modules', 'react-native-web'), // prettier-ignore
  });
```